raiden/network/transport/matrix/client.py

```diff
@@ -262,6 +262,7 @@ def start_listener_thread(self, timeout_ms: int = 30000, exception_handler: Call
         assert not self.should_listen and self.sync_thread is None, 'Already running'
         self.should_listen = True
         self.sync_thread = gevent.spawn(self.listen_forever, timeout_ms, exception_handler)
+        self.sync_thread.name = f'GMatrixClient.listen_forever user_id:{self.user_id}'
```

Comment:
I added these names to aid debugging, without this it's not possible to distinguish greenlets from different apps in a test on a stack trace dump